### ISSUE TITLE
jsx-curly-brace-presence does not complain about JSX tags in braces

### ISSUE THREAD

**Issue by HarrisonB** [yannickcr/eslint-plugin-react#2416]
This should raise an error, right?

```jsx
<div>{ <span></span> }</div>
```

**Comment by ljharb** [comment]
If configured to "never", yes, I would expect so.
